packages/grid/src/ExpandableColumnGridModel.ts

```diff
@@ -0,0 +1,62 @@
+import GridModel from './GridModel';
+import { ModelIndex } from './GridMetrics';
+
+export function isExpandableColumnGridModel(
+  model: GridModel
+): model is ExpandableColumnGridModel {
+  return (model as ExpandableColumnGridModel)?.hasExpandableColumns ?? false;
```

Comment:
This should just check if `hasExpandableColumns !== undefined` - it's just checking if the model is an `ExpandableColumnGridModel` type, which it is even if the value for `hasExpandableColumns` is `false`. When using it, you should be checking `isExpandableColumnGridModel` and then `model.hasExpandableColumns`. I guess we could consolidate that into a function, but it shouldn't be in the type predicate function that narrows the type (IMHO).

Although I guess we've been bitten by this before with `PartitionedGridModel`.